content/en/developers/authorization/oauth2_in_datadog.md

```diff
@@ -28,23 +28,29 @@ This page provides a step-by-step overview on how to implement the OAuth protoco
 
    To learn how to derive the `code_challenge` parameter, see the [PKCE](#authorization-code-grant-flow-with-pkce) section. Your application must save `code_verifier` for the token request in Step 5.
 
-   - In order to build the URL for this post request, use the `site` query parameter that is provided on the redirect to your `onboarding_url`. 
+   - In order to build the URL for this POST request, use the `domain` query parameter that is provided on the redirect to your `onboarding_url`. 
```

Comment:
@bgoldberg122 isn't this a GET request?

Reply:
I think it should be GET. According to the docs that we link [here](https://docs.datadoghq.com/developers/authorization/oauth2_endpoints/?tab=authorizationendpoints#request-authorization-from-a-user)

Reply:
yeah - updating